gcs/go.mod

```diff
@@ -1,6 +1,6 @@
-module github.com/decred/dcrd/gcs/v2
+module github.com/decred/dcrd/gcs/v3
 
-go 1.11
+go 1.13
```

Comment:
This really shouldn't be bumped in this PR since nothing is changing to make the module require it.  I realize that it will ultimately need to be 1.13 with the addition of the `errors.Is` bits, but that is not in this PR.

Reply:
You're right, this was intended for #2262.